opentelemetry-sdk/src/export/logs/mod.rs

```diff
@@ -22,8 +20,8 @@ use std::fmt::Debug;
 #[derive(Debug)]
 pub struct LogBatch<'a> {
     /// The data field contains a slice of tuples, where each tuple consists of a reference to
-    /// a `LogRecord` and a reference to an `InstrumentationLibrary`.
-    data: &'a [(&'a LogRecord, &'a InstrumentationLibrary)],
+    /// a `LogRecord` and a reference to an `InstrumentationScope`.
```

Comment:
nit - change line 14, 18 (and few more) in this file to indicate `instrumentation scope` instead of `instrumentation libraries`.